packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -86,6 +88,7 @@ export function ScreenshotEditorFactory({
     const [croppingRect, setCroppingRect] = hooks.useState<Box>({ startX: 0, startY: 0, endX: 0, endY: 0 });
     const [confirmCrop, setConfirmCrop] = hooks.useState(false);
     const [isResizing, setIsResizing] = hooks.useState(false);
+    const [isCropping, setIsCropping] = hooks.useState(true);
     const [isAnnotating, setIsAnnotating] = hooks.useState(false);
```

Comment:
We might want to make these a single state where the value is more like an enum. 
Right now i'm noticing spots like this:
```
setIsCropping(!isCropping);
setIsAnnotating(false);
```
which will only get more annoying if more tools get added. Not something for now, but when the next one gets added this'll be a fix.